src/software/proto/message_translation/BUILD

```diff
@@ -21,11 +22,35 @@ cc_test(
     name = "tbots_protobuf_test",
     srcs = ["tbots_protobuf_test.cpp"],
     deps = [
-        "//software/proto/message_translation:tbots_protobuf",
+        "tbots_protobuf",
         "//software/test_util",
     ],
 )
 
+cc_library(
+    name = "tbots_geometry",
```

Comment:
Perhaps `convert_geom_to_proto` might be a bit more descriptive? I'm not sure the `tbots_` prefix really adds any value. Similarly for the file names and the test below.

Reply:
I thought the folder name would make it clear, i.e. message_translation for tbots_geometry protos

Reply:
Ah yeah. I agree. This is fine with me.